prow/config/metrics/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I suspect we don't need this. Do we have tests that rely on metrics server? Not a big harm to keep it in though